web/packages/teleterm/src/ui/services/keyboardShortcuts/useKeyboardShortcutFormatters.ts

```diff
@@ -41,18 +41,18 @@ export function useKeyboardShortcutFormatters(): KeyboardShortcutFormatters {
   const keyboardShortcuts = keyboardShortcutsService.getShortcutsConfig();
 
   return {
-    getLabelWithShortcut(label, shortcutKey, options) {
+    getLabelWithShortcut(label, shortcutAction, options) {
```

Comment:
Should those functions be renamed to something like `getLabelWithAccelerator`, `getAccelerator`? I like the action & accelerator distinction, when we say just "shortcut" I feel like we loose the clarity that this distinction brings us.

Reply:
Done! I'm wondering if we should rename the top level `useKeyboardShortcutFormatters`. It feels to me like it is a good, more general name 🤔

Reply:
> I'm wondering if we should rename the top level `useKeyboardShortcutFormatters`. It feels to me like it is a good, more general name 🤔

I feel the same way.